model/post.go

```diff
@@ -31,6 +31,7 @@ type Post struct {
 	Id            string          `json:"id"`
 	CreateAt      int64           `json:"create_at"`
 	UpdateAt      int64           `json:"update_at"`
+	EditAt        int64           `json:"edit_at"`
```

Comment:
Hmmmm, you're right. Alright I guess we need to add the `EditAt` field to do this properly